src/internal/m365/service/groups/enabled.go

```diff
@@ -15,7 +18,20 @@ func IsServiceEnabled(
 	gbi getByIDer,
 	resource string,
 ) (bool, error) {
-	// TODO(meain): check for error message in case groups are
-	// not enabled at all similar to sharepoint
-	return true, nil
+	resp, err := gbi.GetByID(ctx, resource)
+	if err != nil {
+		return false, clues.Wrap(err, "getting group").WithClues(ctx)
+	}
+
+	// according to graph api docs: https://learn.microsoft.com/en-us/graph/api/resources/group?view=graph-rest-1.0
+	// "If the collection contains Unified, the group is a Microsoft 365 group;
+	// otherwise, it's either a security group or distribution group."
+	//
+	// Basically, if it's "unified", then we actually have data to back up.
+	// If it's not unified, then its purely a mailing list, and has no backing data.
```

Comment:
Are we going to have to update this if we add support for other teams/groups features? If we will, could you please add a TODO

Reply:
Nope, shouldn't require any updates.  This is, as far as I can tell, the sole flag for describing backup capability in a group.